src/c++/perf_analyzer/concurrency_worker.h

```diff
@@ -91,7 +93,15 @@ class ConcurrencyWorker : public LoadWorker {
   {
   }
 
-  void Infer() override;
+  virtual void Infer() override;
+
+ protected:
+  bool RunInference();
+
+  void CreateCtxIdTracker();
```

Comment:
Ctx should not be abbreviated

Reply:
Although I agree that method name could be improved, the current PR is merely using this method by inheriting from concurrency worker class.

Reply:
This is a copy and paste. I'd prefer to refactor it in another PR.